cvat-ui/src/components/annotation-page/top-bar/player-buttons.tsx

```diff
@@ -92,23 +141,39 @@ function PlayerButtons(props: Props): JSX.Element {
 
     let nextButton = <Icon className='cvat-player-next-button' component={NextIcon} onClick={onNextFrame} />;
     let nextButtonTooltipMessage = nextRegularText;
-    if (nextButtonType === 'filtered') {
+    if (navigationType === NavigationType.FILTERED) {
         nextButton = (
             <Icon className='cvat-player-next-button-filtered' component={NextFilteredIcon} onClick={onNextFrame} />
         );
         nextButtonTooltipMessage = nextFilteredText;
-    } else if (nextButtonType === 'empty') {
+    } else if (navigationType === NavigationType.EMPTY) {
         nextButton = <Icon className='cvat-player-next-button-empty' component={NextEmptyIcon} onClick={onNextFrame} />;
         nextButtonTooltipMessage = nextEmptyText;
     }
 
+    const disabledStyle: CSSProperties = {
+        pointerEvents: 'none',
+        opacity: 0.5,
+    };
+
     return (
         <Col className='cvat-player-buttons'>
+            <GlobalHotKeys keyMap={subKeyMap} handlers={handlers} />
             <CVATTooltip title='Go to the first frame'>
-                <Icon className='cvat-player-first-button' component={FirstIcon} onClick={onFirstFrame} />
+                <Icon
+                    style={workspace === Workspace.SINGLE_SHAPE ? disabledStyle : {}}
```

Comment:
The same thing with `workspace === Workspace.SINGLE_SHAPE` in this file

Reply:
Done